configuration/pih/pih-config-haiti-hiv.json

```diff
@@ -61,44 +50,8 @@
   "dispositionConfig": "file:configuration/pih/pih-dispositions-haiti.json",
 
   "registrationConfig": {
-    "allowUnknownPatients": false,
-    "allowManualEntryOfPrimaryIdentifier": true,
-    "afterCreatedUrl": "mirebalais/patientRegistration/afterRegistration.page?patientId={{patientId}}&encounterId={{encounterId}}",
-    "matchingPatientsPropertiesToDisplay": [
-      "personName",
-      "gender",
-      "birthdate",
-      "personAddress",
-      "attributeMap['First Name of Mother']",
-      "attributeMap['Telephone Number']"
-    ],
-    "identifierTypesToDisplay": [
-      "a541af1e-105c-40bf-b345-ba1fd6a59b85",
-      "e66645eb-03a8-4991-b4ce-e87318e37566",
-      "9dbea4d4-35a9-4793-959e-952f2a9f5347",
-      "139766e8-15f5-102d-96e4-000c29c2a5d7",
-      "3B954DB1-0D41-498E-A3F9-1E20CCC47323",
-      "d9009bd0-eac9-11e5-a837-0800200c9a66",
-      "b5d0a5bd-adf3-4fe0-a231-5a488f6d2c61",
-      "e797f826-8e8f-11e7-bb31-be2e44b06b34",
-      "e797face-8e8f-11e7-bb31-be2e44b06b34"
-    ],
-    "maxPatientMatchResults": 50,
-    "similarPatientsSearch": {
-      "addressFields": {
-        "address3": "1",
-        "address1": "1"
-      },
-      "personAttributeTypes": {
-        "First Name of Mother": "3",
-        "Telephone Number": "20"
-      }
-    },
-    "demographics": {
-      "mothersName": {
-        "required": true
-      }
-    }
+    "afterCreatedActions": [
+    ]
```

Comment:
The registration for the HIV server is exactly the same as the regular Haiti registration, except that we don't want to create a dossier number... if you look in the main pih-config-haiti it has a section:

`   "afterCreatedActions": [
          "bean:assignDossierNumber"
        ],`

This overrides that, so that no actions are triggered after a patient is created.